corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -71,6 +71,11 @@ hqDefine("app_manager/js/details/screen", function () {
         });
         self.persistCaseContext = ko.observable(detail.persist_case_context || false);
         self.persistentCaseContextXML = ko.observable(detail.persistent_case_context_xml || 'case_name');
+
+        self.caseTileGrouped = ko.observable(detail.case_tile_group.index_identifier || false);
```

Comment:
nit: I don't see an issue with this as is, but should we store  `true` for `caseTileGrouped`  instead of just the `index_identifier` value

Reply:
f344adac247

Reply:
that's cool, i didn't know `!!` could be used like that. Do we still need `|| false`? I think the `!!` handles that already